README.md

```diff
@@ -544,6 +544,9 @@ er:
   # ER diagram (png/jpg) font (font name, font file, font path or keyword)
   # Default is "" ( system default )
   font: M+
+  # ER diagram dot format template, if specified this template file will be used
+  # in place of the defualt one to generate the dot file for graphviz
+  # template: schema.dot.tmpl
```

Comment:
Could you change to new syntax?

Reply:
I forgot that commit during rebase, I'll remove it and keep the changes to the README in one place.